src/data/community-events.json

```diff
@@ -408,6 +408,16 @@
     "startDate": "2024-9-20",
     "endDate": "2024-9-22",
     "imageUrl": "https://ethglobal.com/og.png"
+  },
+   {
+    "title": "ETHMilan",
+    "to": "https://ethmilan.xyz/",
+    "location": "Milan, Italy",
+    "description": "ETMilan returns for its second edition – join us for two days of networking, Ethereum talks & workshops, and an exciting Creative Hackathon!",
+    "startDate": "2023-10-05",
```

Comment:
@gguideth Hi, `startDate` and `endDate` values seem to be wrong, thanks

Reply:
snap, they are – let me fix this, sorry!